public/app/features/logs/components/LogDetailsRow.tsx

```diff
@@ -160,14 +179,32 @@ class UnThemedLogDetailsRow extends PureComponent<Props, State> {
     showFieldsStats: false,
     fieldCount: 0,
     fieldStats: null,
+    showLineScroll: false,
   };
 
+  logRowRef: React.RefObject<HTMLTableCellElement> = React.createRef();
+
   componentDidUpdate() {
     if (this.state.showFieldsStats) {
       this.updateStats();
     }
   }
 
+  componentDidMount(): void {
```

Comment:
If we like this approach I can add in the functionality where it adjusts this property on pane resize. This would probably be necessary when split resize open/closes for clicking on a correlation.